apps/authentication-portal/src/main/webapp/totp.jsp

```diff
@@ -45,8 +45,20 @@
 
     if (Boolean.parseBoolean(request.getParameter(Constants.AUTH_FAILURE))) {
         authenticationFailed = "true";
-
-        if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
+        String errorCode = request.getParameter("errorCode");
+
+        if ( errorCode != null) {
+            if (errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE)) {
+                String lockedReason = request.getParameter("lockedReason");
+                if (lockedReason!=null) {
+                    if (lockedReason.equals("MAX_TOTP_ATTEMPTS_EXCEEDED")) {
```

Comment:
Same like earlier, Wouldn't it will be better to get value `MAX_TOTP_ATTEMPTS_EXCEEDED` and `ADMIN_INITIATED` from a constant

Reply:
Addressed similar cases in 39aa2236f7ad7514ffc41eb777a9ed32f954d76f